src/std/math.earl

```diff
@@ -42,3 +44,102 @@ module Math
     }
     return x;
 }
+
+# Statistical Functions
+
+# Mode
+# This function returns a list of all modes that appear at the
+# highest frequency in the given list in the order modes are found.
+@pub fn mode(lst) {
+    
+    if len(lst) == 0 {
+        return none;
+    }
+
+    # if len(lst) == 1 {
+    #     return lst[0];
+    # }
+    
+    let count = 0;
```

Comment:
What I am about to say is completely unnecessary, but you can now do `let count, max_count, lst_mode = (0, 0, []);` :)